syntax = "proto3";

package homework7;

message FullName {
    string first_name = 1;
    string second_name = 2;
    optional string third_name = 3;
}

message Student {
    FullName full_name = 1;
    repeated int32 marks = 2;
    double average_mark = 3;
}

message StudentsGroup {
    repeated Student students = 1;
}